packages/footer/src/js/react.js

```diff
@@ -0,0 +1,62 @@
+/*! [replace-name] v[replace-version] */
+/***************************************************************************************************************************************************************
+ *
+ * footer function
+ *
+ * Footers help users who reach the bottom of a page without finding what they want.
+ *
+ **************************************************************************************************************************************************************/
+
+import React from 'react';
+import PropTypes from 'prop-types';
+
+
+/**
+ * A section for the footer that contains navigational elements
+ *
+ * @param  {node} children - The inside of this section
+ */
+export const FooterNav = ({ children }) => (
+	<nav className="uikit-footer__navigation">
+		{ children }
+	</nav>
+);
+
+FooterNav.propTypes = {
+	children: PropTypes.node.isRequired,
```

Comment:
A footer without content is literally an empty box with background color. I would require it as it's useless empty